src/pyhf/cli/cli.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+import click
+
+from ..version import __version__
+from . import rootio, spec, stats
+
+logging.basicConfig()
+log = logging.getLogger(__name__)
+
+
+@click.group(context_settings=dict(help_option_names=['-h', '--help']))
+@click.version_option(version=__version__)
+def pyhf():
+    pass
+
+
+# pyhf.add_command(rootio.cli)
```

Comment:
These comments can get deleted now?
```suggestion
```

Reply:
depends on what we want to do with the api. If we want `pyhf rootio xml2json` and `pyhf rootio json2xml` etc...

Reply:
You're saying that if we do want 

```
$ pyhf rootio xml2json
```

then we remove the comment and go with

```python
pyhf.add_command(rootio.json2xml)
pyhf.add_command(rootio.xml2json)
```

but if we would prefer a CLI like 

```
$ pyhf xml2json
```

then we only have

```python
pyhf.add_command(rootio.cli)
```

yes?

If so, then FWIW, I would put my vote behind `pyhf rootio xml2json` to keep the cli from returning a huge number of results when someone tab completes. The grouping should help with that, and I think it is cleaner.

Reply:
Other way around.

`pyhf.add_command(rootio.cli)` adds the `pyhf rootio` -- while `pyhf.add_command(rootio.json2xml)` adds the `pyhf json2xml`.

I haven't gotten autocomplete set up on my end ,but it would be nice to have with Click + pyhf...

Reply:
Ah okay &mdash; sorry. Yeah, since you're preserving the current behavior with how you currently have it I see why you have those lines commented hinting at what would be to come.

Since moving to `pyhf.add_command(rootio.cli)` would be an API change maybe that should happen in a separate `feat` PR?

Reply:
Yep, which is why it's commented out for now (and the tests need to be updated to use the right command).